form-flow/src/main/kotlin/com/ritense/formflow/autoconfigure/FormFlowAutoConfiguration.kt

```diff
@@ -79,9 +80,10 @@ class FormFlowAutoConfiguration {
     fun formFlowService(
         formFlowDefinitionRepository: FormFlowDefinitionRepository,
         formFlowInstanceRepository: FormFlowInstanceRepository,
-        expressionProcessorFactory: ExpressionProcessorFactory
+        expressionProcessorFactory: ExpressionProcessorFactory,
```

Comment:
The expressionProcessorFactory doesn't seem to be used in creating the instance of FormFlowService so it can be removed